src/core/content/connector.js

```diff
@@ -661,11 +669,27 @@ function BaseConnector() {
 	 */
 	this.reactorCallback = null;
 
+	/**
+	 * Used to determine whether or not podcasts should be scrobbled in onStateChanged()
+	 * @type {Boolean}
+	 */
+	let shouldScrobblePodcasts = true;
+	Util.getOption(Options.SCROBBLE_PODCASTS).then((sp) => shouldScrobblePodcasts = sp);
```

Comment:
Two problems here.

1. `Options` is not available in this file at the moment. Not sure if I need to do the giant `define` wrap which is in the other places the options file is `require`d or what.
2. `Util.getOption` is async. This wasn't a problem in the connector since we could write an async function and call it at the top. Here we can't add an `await` without converting the whole `BaseConnector` function async. I believe that this is equivalent to the Google Play behavior of before (initial connector load would include the default value).

Reply:
These two issues will be resolved if we move the check to the core.